docs/source/index.rst

```diff
@@ -1,29 +1,88 @@
 ..  documentation master file
 
-Welcome to Cibyl documentation
-==============================
+=====
+Cibyl
+=====
+
+Cibyl is a command-line interface and REST API for querying CI environments and systems.
+
+Currently it supports out-of-the-box the following CI systems::
```

Comment:
Here I would remove the code block ("::") so that it's parsed as a bullet list, IMO it looks better